src/rbd_fuse/rbd-fuse.cc

```diff
@@ -534,6 +535,31 @@ rbdfs_create(const char *path, mode_t mode, struct fuse_file_info *fi)
 	return r;
 }
 
+int rbdfs_rename(const char *path, const char *destname)
+{
+    const char *srcname = NULL;
+    const char *extra[] = {" ", "@", "/"};
```

Comment:
I don't think there is any restriction for spaces in the image name -- at least the rbd CLI doesn't restrict it.